### PR TITLE
Summit navbar [CI-STABLE][RHIF-37]

### PR DESCRIPTION
**Pull request by adonispuente**
merged into ci-beta, merging now into ci-stable. Navbar should look like this 
image

### CODE REVIEW

File: chrome/rhel-navigation.json [hunk 1]
```diff
@@ -1,301 +1,276 @@
...
+                "href": "/insights/compliance/systems",
+                "product": "Red Hat Insights"
+              }
             ]
-        },
-        {
-            "title": "Product Materials",
+          },
+          {
+            "title": "Malware",
```
Comment (Hyperkid123) [inline chrome/rhel-navigation.json]: Malware is not in the stable mock

File: chrome/rhel-navigation.json [hunk 2]
```diff
@@ -1,301 +1,276 @@
...
+                "title": "Documentation",
+                "isExternal": true,
+                "filterable": false,
+                "href": "https://access.redhat.com/products/subscription-central"
+              }
+            ]
+          },
+          {
+            "appId": "image_builder",
+            "title": "Image builder",
```
Comment (Hyperkid123) [inline chrome/rhel-navigation.json]: Image builder is not on the stable mock

**Review by Hyperkid123 — COMMENTED** [review]
I see some items that are not on the mockup.

**Comment by adonispuente** [comment]
Good catch @Hyperkid123 , I hadnt fully understood how each branch worked. I thought the values showed depending on a key/value pair.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**